package.json

```diff
@@ -10,7 +10,7 @@
     "lint:fix": "eslint . --fix",
     "unit": "SQS_AUDIT_EVENT_QUEUE_URL=queueUrl jest --testPathIgnorePatterns=\"integration-tests/\"",
     "test": "npm run unit --",
-    "test:coverage": "npm run unit -- --coverage",
+    "test:coverage": "npm run unit && npm run test:infra -- --coverage",
```

Comment:
Rather than make this part of coverage can we update the check-pr action to run infra tests as it own action?

Reply:
yeah sure, ill make that change to the workflow and update the PR, thanks Jamie

Reply:
@gds-jbentham PR has been updated to run the infra tests in their own action on the run-unit-tests.yml workflow file